Controller/AssetCacheKeyFilter.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/*
+ * This file is part of the Assetic package, an OpenSky project.
+ *
+ * (c) 2010-2014 OpenSky Project Inc
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\AsseticBundle\Controller;
```

Comment:
We didn't give a lot of consideration to where this file should go. Probably not Controller. Hopefully someone can chime in with a suggestion here